### ISSUE TITLE
Dashboard

### ISSUE THREAD

**Issue by asakpke** [opensourcepos/opensourcepos#3012]
### Background information
- [x] Reporting an issue of an unmodified OSPOS installation
- [x] Checked [open and closed issues](https://github.com/opensourcepos/opensourcepos/issues?utf8=%E2%9C%93&q=is%3Aissue) and no similar issue was already reported (please make sure you searched!)
- [x] Read [README](https://github.com/opensourcepos/opensourcepos/blob/master/README.md), [WHATS_NEW](https://github.com/opensourcepos/opensourcepos/blob/master/WHATS_NEW.txt), [INSTALL.md](https://github.com/opensourcepos/opensourcepos/blob/master/INSTALL.md) and [UPGRADE](https://github.com/opensourcepos/opensourcepos/blob/master/UPGRADE.txt)
- [x] Read the [FAQ](https://github.com/opensourcepos/opensourcepos#faq) for any known install and/or upgrade gotchas (in specific PHP extensions installed)
- [x] Read the [wiki](https://github.com/opensourcepos/opensourcepos/wiki)
- [x] Executed any database upgrade scripts if an upgrade pre 3.0.0 (e.g. database/2.4_to_3.0.sql)
- [x] Aware the installation code is in [bintray](https://bintray.com/jekkos/opensourcepos/opensourcepos/view/files?sort=updated&order=asc#files) (see README), and [GitHub master](https://github.com/opensourcepos/opensourcepos/tree/master) is for [developers only](https://github.com/opensourcepos/opensourcepos/wiki/Development-setup) and therefore not complete nor stable

### Installation information

- OSPOS version is: 3.3.2
- OSPOS git commit hash is: bb309c
- PHP version is: 7.4.3
- MySQL or MariaDB version is: 8.0.22
- OS and version is: Linux Mint 20
- WebServer is: Apache
- Selected language is: 
- (If applicable) Docker installation: 
- (If applicable) Installation package for the LAMP/LEMP stack is: LAMP

### New Feature

Have anybody thought/planned/done something about create a dashboard instead of showing icons on the home page?

For example:

![this one](https://www.clicdata.com/wp-content/uploads/2019/06/example-dashboard-014-1-931x1024.png)

**Comment by jekkos** [comment]
I agree, something like this would be nice. I feel it should be designed in a generic way so people can choose which reports to display.. for example by embedding some of the graphical reports in a tiled setup and have this configured through global config.

**Comment by asakpke** [comment]
I'm willing to work but I have not worked on v3 yet and I have available in weekends. Since 2015, a lot of things are changed. I need guidance, feedback, help etc.

**Comment by jekkos** [comment]
I think at the time @daN4cat had some nice ideas in using reportico for this. #1433 . I think he did some work using that tool outside of ospos.

**Comment by atifbuttonline** [comment]
I am also interested in this DASHBOARD-Issue

**Comment by asakpke** [comment]
I'm running Free Web Development (FWD) sessions online weekly, Saturday 10 PM to 12 AM (Pakistan standard time GMT+5) https://meet.google.com/bgj-kjcs-vwq 📞 03005393652 Videos ▶️/asakpke. 

My plan is to continue OSPOS development in my FWD sessions & grab IT literates involved in OSPOS development. This session is mainly in Urdu/Hindi language but English speakers are welcome to join & we will switch to English. Following are the details of last session, if you are interested.

Here's the video of last session "21-Nov-20 FREE Web Development session" https://youtu.be/x8V6B0luTJc

For more videos about FREE Web Development Session, check this playlist "FREE Web Development Sessions" https://www.youtube.com/playlist?list=PLuSlqiqQkM0wepYUxRHnZVdeQvVOcwP9f

Timestamp
00:00:00 Intro in English
00:00:55 Intro in Urdu/Hindi
00:04:46 HTML Multimedia
00:05:00 What's API (application programming interface)?
00:07:43 What's SDK (software development kit)?
00:09:05 Question: Can anybody read our data using API path/calls?
00:12:50 Question: How to login web app that is using MD5 password?
00:13:25 How to break MD5 password?
00:16:10 Why use MD5 password?
00:17:08 How to use MD5 password?
00:17:50 How to break MD5 password?
00:19:46 Answer: How to login web app that is using MD5 password?
00:19:58 How to confirm if MD5 password is being used?
00:39:55 HTML Multimedia
00:44:00 HTML Video
00:56:07 HTML Audio
00:57:50 HTML Plug-ins
01:00:02 HTML YouTube Videos
01:03:15 How to hide YouTube title/logo? (we tried but now they don't support)?
01:33:00 HTML YouTube Videos
01:36:00 OSPOS
01:37:43 OSPOS issue
01:38:08 Question: How to use open source project & fix issues?
01:48:20 Open source development plan
01:49:26 OSPOS setup help on Windows
01:53:43 Question: who's the owner of OSPOS?
01:56:00 Shortcut keys on OSPOS (not working now or some issue)
01:57:35 Customize OSPOS
02:00:46 OSPOS themes
02:05:36 Earnings from open source project i.e OSPOS
02:06:15 How open source software are growing?
02:08:36 Most interesting thing in open source software (develop to gather & earn)
02:08:43 Who's developing Linux OS (an open source project)?
02:09:34 Linux is used in Android & Chrome OS (Chromebooks)
02:10:01 How developer can earn on GitHub?
02:10:27 How to get job from GitHub?

WhatsApp Group: to engage members & share session details.
https://chat.whatsapp.com/CXHWS94r7A913ikl7eHrra

More info: https://www.facebook.com/FreeWebDevelopmentOnline

**Comment by TouchstoneTheGreat** [comment]
@asakpke invite expired

**Comment by asakpke** [comment]
> @asakpke invite expired

Here are new links 
- [WhatsApp](https://chat.whatsapp.com/EhEIPA7HdKb6MnvlzwyqLa)
- [Signal app](https://signal.group/#CjQKIIssIuNAzXUTNutw2TIOPso2Hw3W11Lc6nFELTzPQQHSEhD4buQi25ABWVuAyoZKfevd)
